client/src/tests/procedure_tests.rs

```diff
@@ -804,3 +804,51 @@ async fn test_bip39_recover_zeroize() -> Result<(), Box<dyn std::error::Error>>
 
     Ok(())
 }
+
+#[test]
+#[cfg(feature = "insecure")]
+fn test_usecase_concatkdf() {
+    use crate::procedures::ConcatSecret;
+
+    let client_path = b"client-path";
+
+    let location_a = Location::const_generic(b"first-loc".to_vec(), b"first-loc".to_vec());
+    let location_b = Location::const_generic(b"second-loc".to_vec(), b"second-loc".to_vec());
+    let output_location = Location::const_generic(b"output-loc".to_vec(), b"output-loc".to_vec());
+
+    let stronghold = Stronghold::default();
+    let client = stronghold.create_client(client_path).expect("Could not create client");
```

Comment:
Nit: Just FYI, there is now a [recommended message style](https://github.com/rust-lang/rust/blob/96b9bb4620f4d48aa25c381c7ea77e0cab48ac5b/library/core/src/result.rs#L1036) for `expect` messages.

Reply:
Thanks for pointing that out. Makes more sense indeed :)